src/main/java/uk/gov/hmcts/reform/sscs/service/venue/VenueRpcDetailsService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.sscs.service.venue;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+import uk.gov.hmcts.reform.sscs.service.VenueDataLoader;
+
+import java.util.Optional;
+
+@Service
+@Slf4j
+public class VenueRpcDetailsService {
+
+    private final VenueDataLoader venueDataLoader;
+
+    @Autowired
+    public VenueRpcDetailsService(VenueDataLoader venueDataLoader) {
```

Comment:
Minor/Optional: we can use lombok `@RequiredArgsConstructor` for this.. also no `@Autowired` required.

Reply:
will do